### PR TITLE
Update Kueue's manifests to 0.6.1

### PR DESCRIPTION
**Pull request by PBundyra**

### CODE REVIEW

File: tutorials-and-examples/workflow-orchestration/dws-examples/kueue-manifests.yaml [hunk 1]
```diff
@@ -11052,7 +11768,7 @@ spec:
         - --feature-gates=ProvisioningACC=true
         command:
         - /manager
-        image: registry.k8s.io/kueue/kueue:v0.5.2
```
Comment (alculquicondor) [inline tutorials-and-examples/workflow-orchestration/dws-examples/kueue-manifests.yaml:11055]: registry.k8s.io/kueue/kueue:v0.6.1

**Comment by andrewsykim** [comment]
Skipping CI checks since it doesn't test the Kueue examples/tutorials

**Pull request merged**